pydruid/utils/query_utils.py

```diff
@@ -28,9 +30,13 @@ def __init__(self, f, dialect="excel-tab", encoding="utf-8", **kwds):
         self.writer = csv.writer(self.stream, dialect=dialect, **kwds)
         self.encoder = codecs.getincrementalencoder(encoding)()
 
-    @staticmethod
-    def __encode(data):
-        return str(data) if isinstance(data, str) else data
+    def __encode(self, data):
+        data = str(data) if isinstance(data, six.integer_types) else data
+        if not six.PY3:
+            data = data.encode("utf-8") if isinstance(data, unicode) else data  # noqa
+            data = data.decode("utf-8")
```

Comment:
Why twice? Seems wrong.

Reply:
It's doing an `encode` and then a `decode`. Why is that, I'm not sure. I copied it from the `0.5` branch: https://github.com/druid-io/pydruid/blob/0.5/pydruid/utils/query_utils.py#L35-L36 Since we are not going to use this (it's just meant for exporting `tsv` files and we are only going to use the DBAPI part), I didn't bother to see how it could be improved. In the `upstream` master branch this code does no longer exist either way.